tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -272,6 +270,7 @@ install_cc_tee_kernel() {
 		"${final_tarball_path}" \
 		&& return 0
 
+	info "build initramfs for tee kernel"
```

Comment:
```suggestion
	info "build initramfs for TEE kernel"
```

(Capitalizing acronyms)